content/master/concepts/_index.md

```diff
@@ -55,6 +55,16 @@ building and managing external resources through Kubernetes.
   Composite Resource. Platform users create Claims in their unique namespace,
   isolating their resources from other teams in other namespaces. 
 
+* [**Composition Functions**]({{<ref "./composition-functions">}}) are custom
+  programs, written your programming language of choice, to apply logic and
+  loops before or after Crossplane creates resources. 
+
+* [**Patches and Transforms**]({{<ref "./patch-and-transform">}}) allow platform
+  engineers to use user inputs to their custom API and change how Crossplane
+  creates resources. Patches and transforms allow for flexible and
+  abstract inputs like `big` or `encrypted` to have specific meanings when
+  creating the actual managed resources.
+
```

Comment:
This content was removed in https://github.com/crossplane/docs/pull/799/files#diff-b256b27bd644a45659738110dfece51b3c4961f7ca2cdf9913524c04b0a54623 with @negz's big PR to make docs "functions first".

Do we understand how this is getting added back in here? I'm concerned about the validating of the other changes in this massive PR if it's accidentally restoring content that was intentionally removed...can we get to the bottom of this? 🙏